apps/admin/types/dtos/announcement.ts

```diff
@@ -1,6 +1,11 @@
+import type { StudyApiResponseV2Dto } from "./studyList";
+
 export interface AnnouncementApiResponseDto {
-  studyAnnounceId: number;
-  title: string;
-  link: string;
-  createdDate: string;
+  study: StudyApiResponseV2Dto;
+  studyAnnouncement: {
+    studyAnnounceId: number;
```

Comment:
p2;

```suggestion
    studyAnnouncementId: number;
```
![image](https://github.com/user-attachments/assets/e379bdf2-7f84-431e-88b9-d1d2bb3cf715)
스웨거 상에서는 studyAnnouncementId 로 되어있어서 undefined 로 들어간 거 같아요! 요기만 수정해주면 수정/삭제도 잘 동작할 거 같습니당